modules/mono/editor/GodotTools/GodotTools/Ides/Rider/RiderPathManager.cs

```diff
@@ -9,6 +9,13 @@ namespace GodotTools.Ides.Rider
 {
     public static class RiderPathManager
     {
+        static readonly RiderPathLocator RiderPathLocator;
```

Comment:
We prefer to be explicitly about the member accessibility.

```suggestion
        private static readonly RiderPathLocator RiderPathLocator;
```